script.js

```diff
@@ -17,4 +17,122 @@ crossMenu2.addEventListener('click', closeMobMenu);
+  const textCont = document.createElement('div');
+  const textBtnCont = document.createElement('div');
+  const textBtn1 = document.createElement('button');
+  const textBtn2 = document.createElement('button');
+  const btnLink1 = document.createElement('a');
+  const btnLink2 = document.createElement('a');
+  const btnImg1 = document.createElement('img');
+  const btnImg2 = document.createElement('img');
+
+  contPop.classList.add('cont-pop-up');
+  section.id = ('modal-pop-up');
+  section.appendChild(contPop);
+  section.classList.add('modal-pop-up');
+  Body.appendChild(section);
+  contPop.appendChild(header1);
+  header1.appendChild(heading);
+  header1.appendChild(closeButton);
+  contPop.appendChild(projectLang);
+  projectLang.appendChild(projectLangList1);
+  projectLang.appendChild(projectLangList2);
+  projectLang.appendChild(projectLangList3);
+  contPop.appendChild(popCont);
+  popCont.appendChild(modalImg);
+  popCont.appendChild(textCont);
+  textCont.appendChild(projectText);
+  textCont.appendChild(textBtnCont);
+  textBtnCont.appendChild(textBtn1);
+  textBtnCont.appendChild(textBtn2);
+  textBtn1.appendChild(btnLink1);
+  textBtn2.appendChild(btnLink2);
+
+  header1.classList.add('pop-cont-container');
+  heading.classList.add('pop-heading-det');
+  projectText.classList.add('pop-text-det');
+  projectLang.classList.add('pop-ul-det');
+  projectLangList1.classList.add('pop-li-det');
+  projectLangList2.classList.add('pop-li-det');
+  projectLangList3.classList.add('pop-li-det');
+  closeButton.dataset.modalClose = '';
+  closeButton.classList.add('close-button');
+  modalImg.classList.add('modal-pop-img');
+  popCont.classList.add('modal-cont-con');
+  textCont.classList.add('text-cont-pop');
+  textBtnCont.classList.add('text-btn-con');
+  textBtn1.classList.add('btn-modal');
+  textBtn2.classList.add('btn-modal');
+  btnLink1.classList.add('modal-btn');
+  btnLink2.classList.add('modal-btn');
+  btnImg1.classList.add('btn-img');
+  btnImg2.classList.add('btn-img');
+
+  heading.textContent = modal1.title;
+  projectText.textContent = modal1.projectText;
+  closeButton.innerHTML = '&times;';
+  projectLangList1.textContent = modal1.language1;
+  projectLangList2.textContent = modal1.language2;
+  projectLangList3.textContent = modal1.language3;
+  modalImg.src = 'Images/SnapshootPortfolio.png';
+  btnLink1.textContent = 'See Live';
+  btnLink1.href = modal1.live;
+  btnLink1.appendChild(btnImg1);
+  btnImg1.src = 'Images/Icon-Export.png';
+  btnLink2.textContent = 'See Source';
+  btnLink2.href = modal1.source;
+  btnLink2.appendChild(btnImg2);
+  btnImg2.src = 'Images/github.png';
+
+  const closeModal = document.querySelectorAll('[data-modal-close]');
+
+  function closePopUp(popUpMod) {
+    popUpMod.classList.remove('active');
+  }
+
+  closeModal.forEach((button) => {
+    button.addEventListener('click', () => {
+      const popUpMod = button.closest('.modal-pop-up');
+      closePopUp(popUpMod);
+    });
+  });
```

Comment:
- [ ] For this project, the JavaScript code should implement an array of objects to store JavaScript objects containing the information for each project.
- [ ] The above requirements means that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript**. 
- [ ] Modal popup data is populated with the data from the projects array.
- [ ] All these requirements will make adding projects in the future easy. You will only add it in one place - as another object in the projects array.